### ISSUE TITLE
ImportError: cannot import name 'ALGENCAN'

### ISSUE THREAD

**Issue by artie-l** [madebr/pyOpt#9]
Hello, 

I succeded to install pyOpt on Python 3.6 and 3.7, but when I try to run the code, I get this error: 
```
Error importing pyopt
Traceback (most recent call last):

  File "C:\Users\**\Desktop\test.py", line 48, in <module>
    from pyOpt import Optimization   ## Library for optimization

  File "C:\Users\Artem\anaconda3\envs\py36\lib\site-packages\pyOpt\__init__.py", line 10, in <module>
    from .pyALGENCAN import ALGENCAN

ImportError: cannot import name 'ALGENCAN'
```

Maybe someone encountered this problem?

**Comment by madebr** [comment]
I just ran the following commands on master with success:
```sh
#change directory to pyOpt
cd pyOpt

# create venv
python -m venv myvenv

# activate venv
source myvenv/bin/activate

# install numpy
python -m pip install numpy

# install pyopt
python setup.py install

# run an example
python examples/rosenbrock.py
```

**Comment by madebr** [comment]
I just ran the following commands on master with success:
```sh
#change directory to pyOpt
cd pyOpt

# create venv
python -m venv myvenv

# activate venv
source myvenv/bin/activate

# install numpy
python -m pip install numpy

# install pyopt
python setup.py install

# run tp037 examples which uses ALGENCAN solver
python examples/tp037.py
```

**Comment by madebr** [comment]
But indeed.
Running a simple
```python
import pyOpt
```
throws this `ImportError`.

This should be fixed in 86c585de117db880d0d29b36cc37c006940b3528

**Comment by artie-l** [comment]
Works great now, thanks!
